src/main/resources/application.yaml

```diff
@@ -135,11 +135,11 @@ security:
     response:
       enable: true
 
-identity-account-mapper:
-  hostname : "https://identity.sandbox.mifos.io"
+identity_account_mapper:
+  hostname : "https://identity-mapper.sandbox.fynarfin.io"
 
-bulk-processor:
-  hostname : "https://bulk-connector.sandbox.mifos.io"
+bulk_processor:
+  hostname : "https://bulk-connector.sandbox.fynarfin.io"
```

Comment:
push service names to the application yaml so that by default these can work within the cluster